code/game/machinery/doors/airlock.dm

```diff
@@ -552,7 +552,7 @@ About the new airlock wires panel:
 		var/mob/living/carbon/human/H = user
 		if(H.getBrainLoss() >= 60)
 			playsound(src.loc, 'sound/effects/bang.ogg', 25, 1)
-			if(!istype(H.head, /obj/item/clothing/head/helmet))
+			if(!(istype(H.head, /obj/item/clothing/head/helmet || istype(/obj/item/clothing/head/hardhat)))
```

Comment:
`istype(/obj/item/clothing/head/hardhat)` should be `istype(H.head, /obj/item/clothing/head/hardhat)`